services/QuillLMS/app/queries/prompt_feedback_history.rb

```diff
@@ -18,36 +25,126 @@ def self.promptwise_sessions(activity_id)
 
     def self.promptwise_postprocessing(grouped_feedback_histories)
         prompt_hash = {}
-
+        
         prompts = Comprehension::Prompt.where(id: grouped_feedback_histories.map(&:prompt_id))
    
         grouped_feedback_histories.each do |prompt_session|
-            prompt_id = prompt_session.prompt_id
+            prompt_id = prompt_session.prompt_id.to_s.to_sym
 
             prompt_hash[prompt_id] ||= {
-                optimal_final_attempts: 0, 
-                session_count: 0, 
-                total_responses: 0,
-                final_attempt_pct_optimal: 0,
-                final_attempt_pct_not_optimal: 0,
-                display_name: ''
+
+                total_responses: 0.0,
```

Comment:
I've standardized the order of all these metrics everywhere, for legibility

Reply:
I'm a big fan of this kind of standardization.  I wonder if we can come up with a good general rubric to use for this kind of thing moving forward, or if we'll just have to do it case-by-case.